pkg/client/client.go

```diff
@@ -187,7 +203,20 @@ func getExecCredentialWithToken(token, expirationTimestamp string) (*datamodel.E
 	}, nil
 }
 
-func loadExecCredential() (*datamodel.ExecCredential, error) {
+type ClientHelpers interface {
```

Comment:
I think in this case we'd only actually need to mock `getTLSConfig` since by default it uses the `os` package to interact with the underlying FS - can we rename the interface differently to reflect that?